WooCommerce/Classes/ViewRelated/Dashboard/DashboardView.swift

```diff
@@ -162,6 +165,7 @@ private extension DashboardView {
                     }, onViewAllAnalytics: { siteID, siteTimeZone, timeRange in
                         onViewAllAnalytics?(siteID, siteTimeZone, timeRange)
                     })
+                    .padding(.vertical, Layout.padding)
```

Comment:
Instead of adding paddings to the cards, we can wrap the `ForEach` inside an explicit `VStack` with the desired spacing.

Reply:
Great idea, thanks, handled in https://github.com/woocommerce/woocommerce-ios/pull/12493/commits/7f970a6e21c0b3c2b057bb687b9bce809dc63ac6